sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -474,6 +496,52 @@ def __repr__(self):
             )[:1024]
 
 
+class FormSelectionMark(FormElement):
+    """Information about the extracted selection mark.
+
+    :ivar str text: The text content - not returned for FormSelectionMark.
```

Comment:
hmm interesting. So the service returns it but we are ignoring it because of `State`?

Reply:
Service doesn't return `text` for a selectionMark on FormPage. I'm confused - are you saying that you set `text` == `state` here?

Reply:
Yes :|

Reply:
My only concern with doing that is if layout ever does start returning the text associated with the selection mark, then we would need to start returning something different here.

Reply:
Hmmm good point. Yeah. Better be safe here. I will remove from .NET.
Thanks!